cmd/cluster.go

```diff
@@ -1069,15 +1069,17 @@ func (c *config) updateCluster(args []string) error {
 		labelMap := current.Labels
 
 		for _, l := range removeLabels {
-			parts := strings.SplitN(l, "=", 2)
-			delete(labelMap, parts[0])
+			key, _, found := strings.Cut(l, "=")
+			if found {
```

Comment:
This changes current behavior because before that a user could remove labels without `=`. IMHO it's cleaner when the error message "provided labels must be in the form..." gets returned when introducing a behavioral change like this.